pom.xml

```diff
@@ -158,6 +158,7 @@
 						<includes>
 							<include>org.junit.platform:*</include>
 							<include>org.junit.jupiter:junit-jupiter-engine</include>
+							<include>org.junit.jupiter:junit-jupiter-params </include>
```

Comment:
I would rather completely remove:
```
<include>org.junit.jupiter:junit-jupiter-engine</include>
<include>org.junit.jupiter:junit-jupiter-params </include>
```
Only `<include>org.junit.platform:*</include>` seems to be really required here. That shading causes some other issues as well (mentioned here: https://github.com/pitest/pitest-junit5-plugin/issues/24)

Reply:
@anatoily-balakirev I though the way it works is shade only filter in the dependencies that are mentioned in the `<includes>`. So if I only do` <include>org.junit.platform:*</include>`, that still won't pull in the parameterized test dependencies which are in `org.junit.jupiter:junit-jupiter-params`